apps/s_server.c

```diff
@@ -149,7 +149,7 @@ static int dtlslisten = 0;
 static int early_data = 0;
 
 #ifndef OPENSSL_NO_PSK
-static const char psk_identity[] = "Client_identity";
+char *psk_identity = "Client_identity";
```

Comment:
yup, i saw that as well, and thought i'd already pushed a fix for that, but found it still in my working copy, not committed yet.  sorry about the delay, i've pushed it now.